openstack/css/v1/clusters/UpdateClusterFlavor.go

```diff
@@ -0,0 +1,54 @@
+package clusters
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/build"
+)
+
+type ClusterFlavorOptsBuilder interface {
+}
+
+type ClusterFlavorOpts struct {
+	// Indicates whether to verify replicas.
+	NeedCheckReplica bool `json:"needCheckReplica"`
+	// ID of the new flavor.
+	NewFlavorID string `json:"newFlavorId" required:"true"`
+}
+
+type ClusterNodeFlavorOpts struct {
+	// Indicates whether to verify replicas.
+	NeedCheckReplica bool `json:"needCheckReplica"`
+	// ID of the new flavor.
+	NewFlavorID string `json:"newFlavorId" required:"true"`
+	// Type of the cluster node to modify.
+	NodeType string `json:"type" required:"true"`
+}
+
```

Comment:
There is no method binding the interface to structs. At this point, there is no use of passing this explicitly. Furthermore, if two APIs are doing exact same thing, I would suggest choosing the one with fewer params or one with finer control. Use only one opts in param.

Reply:
@sattila1999 could you check this in actual doc?